spk/comskip/Makefile

```diff
@@ -0,0 +1,38 @@
+SPK_NAME = comskip
+SPK_VERS = 0.81.089
+SPK_REV = 1
+SPK_ICON = src/comskip.gif
+
+DEPENDS = cross/ffmpeg cross/argtable cross/$(SPK_NAME)
+
+RELOAD_UI = yes
+STARTABLE = no
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Comskip is a free mpeg commercial detector
+DISPLAY_NAME = Comskip
+CHANGELOG = "Initial version"
+
+HOMEPAGE = http://www.kaashoek.com/comskip/
+LICENSE  = GNU/LGPL
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
+
+INSTALL_PREFIX = /usr/local/$(SPK_NAME)
+
+POST_STRIP_TARGET = comskip_extra_install
+
+include ../../mk/spksrc.spk.mk
+
+.PHONY: comskip_extra_install
+comskip_extra_install:
+	install -m 755 -d $(STAGING_DIR)/var
+	install -m 644 src/comskip.ini $(STAGING_DIR)/var/comskip.ini
+	install -m 755 -d $(STAGING_DIR)/app
```

Comment:
L32-L38 can be removed: I don't think those icons are ever used for pure command-line packages. The packaging process takes care of adding two icons which are used for Package Center.